src/app/data/forms/textarea.ts

```diff
@@ -5,6 +5,7 @@ import { FormControl } from './form-control';
 export class Textarea extends FormControl {
   lines = 5;
   wrap = true;
+  value = '';
```

Comment:
Da alle FormControls ein value haben, wäre es am besten, das in die abstrakte Basisklasse aufzunehmen, anstatt in jedem einzelnen FormControl